db/db_impl_write.cc

```diff
@@ -148,8 +169,10 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
   // and protects against concurrent loggers and concurrent writes
   // into memtables
 
-  last_batch_group_size_ =
+  uint64_t batch_group_size =
       write_thread_.EnterAsBatchGroupLeader(&w, &write_group);
+  last_batch_group_size_.fetch_add(batch_group_size,
+                                   std::memory_order_relaxed);
```

Comment:
For 2pc-optmal case, you don't go through throttling logic at all (if I understand correctly), then why  keep this `last_batch_group_size_` updated at all? For non-2pc-optimal case, there is a regression of performance replacing a normal set to an atomic update.